app/models/metadata_presenter/metadata.rb

```diff
@@ -2,10 +2,11 @@ class MetadataPresenter::Metadata
   include ActiveModel::Conversion
   extend ActiveModel::Naming
 
-  attr_reader :metadata
+  attr_reader :metadata, :errors
 
   def initialize(metadata)
     @metadata = OpenStruct.new(metadata)
+    @errors = ActiveModel::Errors.new(self)
```

Comment:
How this error is filled when he validation is triggered?

Reply:
currently it isn't coz we haven't written the `validate!` method yet 😆

Reply:
I see, I understand the slack message now 😅